editor/src/components/canvas/controls/new-canvas-controls.tsx

```diff
@@ -1,3 +1,5 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/react'
```

Comment:
Does this imply that we now need to include these two lines at the top of every `.tsx` file in the editor? :grimacing:

Reply:
If we want to use the magical `css` property, yeah.

Reply:
but this file was not using it, seems like an unused import to me?

Reply:
It's definitely using the `css` property, where it renders the `TextEditor` element.

Reply:
interesting, so this is broken on master I guess

Reply:
> Does this imply that we now need to include these two lines at the top of every .tsx file in the editor? 😬

that was the case before IIRC?